sdk/identity/azure-identity-extensions/Azure-Database-for-PostgreSQL-README.md

```diff
@@ -120,11 +120,11 @@ To use Microsoft Entra access with Azure Database for PostgreSQL, you should set
 Only a Microsoft Entra admin user can create/enable users for Microsoft Entra-based authentication.
 
 ```Azure CLI
-az postgres server ad-admin create \
+az postgres flexible-server ad-admin create \
     --resource-group $AZ_RESOURCE_GROUP \
     --server-name $AZ_DATABASE_NAME \
     --display-name $AZ_POSTGRESQL_AD_ADMIN_USERNAME \
-    --object-id `(az ad signed-in-user show --query id -o tsv)`
+    --object-id $(az ad signed-in-user show --query id)
```

Comment:
Does removing `-o tsv` break this? Perhaps not since the shell consumes the quotes that are added by removing it?